### ISSUE TITLE
Checkbox/Radio size

### ISSUE THREAD

**Issue by deakjahn** [flutter/flutter#81334]
## Use case

The `Checkbox` and `Radio` widget have fixed sizes that might be too small in some cases, also depending on the overall UI design. An UI with usual icon buttons of the standard size might want to use the same size for these controls as well, especially if they appear near to each other.

## Proposal

The suggestion is to add a `scale` variable to both `CheckboxThemeData` and `RadioThemeData` and pass them through to the actual widgets. Individual checkboxes can be manually scaled as:

    Transform.scale(scale: 2, child: Checkbox() ...

but this won't help `CheckboxListTile` where the size remains the same. Changing the underlying widget to use this scaling approach would solve that, too.

**Comment by danagbemava-nc** [comment]
A similar proposal can be found in #98601

**Comment by MitchellGoodwin** [comment]
I put a comment on the PR for the Checkbox, but I'll copy it here:

I'm not sure if we want to add a property to adjust the size of the Checkbox widget itself. The checkbox's dimensions are based on the [Material 3 specs](https://m3.material.io/components/checkbox/specs) which are pretty deliberate on their dimensions. This has come up before, and the guidance we've used in the https://github.com/flutter/flutter/issues/19705#issuecomment-407946711 is to wrap the widget with a Transform similar to how you would use Padding.

All that said, the use case of needing to adjust the size in a CheckboxListTile makes sense to me. I think it would be good if we added a property to that constructor asking for a size adjustment, and internally that widget can wrap it's checkbox with a Transform.
